src/lang/swedish.lng

```diff
@@ -30,11 +30,11 @@ STR_NAME_COAL_MINE                  :Kolgruva
 STR_NAME_OIL_WELLS                  :Oljekälla
 STR_NAME_PAPER_MILL                 :Pappersbruk
 STR_NAME_RECYCLING_CENTER           :Återvinningscentral
-STR_NAME_BUILDING_MATERIALS_FACTORY :Building Materials Factory
-STR_NAME_FOOD_PROCESSING_PLANT      :Food Processing Plant
+STR_NAME_BUILDING_MATERIALS_FACTORY :Byggnadsmaterialsfabrik
+STR_NAME_FOOD_PROCESSING_PLANT      :Livsmedelsanläggning
```

Comment:
This one was tricky, got this from https://www.livsmedelsverket.se/foretagande-regler-kontroll/sa-kontrolleras-ditt-foretag/livsmedelsanlaggningar